### PR TITLE
[wave2-884] Extending the more info brick

### PR DESCRIPTION
**Pull request by mhughes2012**

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_mi/js/ucgy_gbl_mi.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+(function ($) {
```
Comment (nindusaini) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_mi/js/ucgy_gbl_mi.js]: Can we please explain a bit in comments about the need/use of this function?

Comment (mhughes2012) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_mi/js/ucgy_gbl_mi.js]: added a couple of js comments.

**Comment by dpfitzsi** [comment]
Needs a second peer review of probo now that front end has been added.

**Comment by robbdavis** [comment]
Front end checks done. Looks good.

**Comment by balrajb** [comment]
Probo is working now, can't see "Colour picker" in https://eb4a6540-b4be-4b88-b5a5-8c74f9195fc4--pr-508.probo.build/admin/structure/fieldable-panels-panes/view/488/edit

**Comment by balrajb** [comment]
colour picker still not working, Mike is looking into it.

**Pull request merged**